.dockerignore

```diff
@@ -1,17 +1,23 @@
-.github
-.vscode
+.github/
+.vscode/
 node_modules/
-vendor
-.env
-.env.example
+vendor/
+.env*
 .gitignore
 .gitattributes
 .php-cs-fixer.cache
 .prettierignore
 .prettierrc.json
 .rr.yaml
 .styleci.yml
+docker-compose.yml
+Dockerfile
+README.md
 LICENSE
 rr
 database/seeders/tutors.csv
-scaleup
\ No newline at end of file
+charts/
+deploy/
+scaleup/
```

Comment:
```suggestion
```

dont know why this is included herem folder will be deleted with this PR and is not relevant for dockerignore anymore